.github/workflows/visuals.yml

```diff
@@ -1,13 +1,12 @@
 name: Visual Regression Tests
 
 on:
-  workflow_dispatch:
-  # pull_request:
-  #   branches:
-  #     - main
-  # push:
-  #   branches:
-  #     - gh-readonly-queue/main/**
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - gh-readonly-queue/main/**
```

Comment:
This will currently do nothing because we're not using the merge queue feature right now. But in case we do use it again in the future, this will just worked :)